src/detection_win.cpp

```diff
@@ -423,6 +424,21 @@ void ExtractDeviceInfo(HDEVINFO hDevInfo, SP_DEVINFO_DATA* pspDevInfoData, TCHAR
 		// Use this to extract VID / PID
 		extractVidPid(buf, resultItem);
 	}
+
+	DWORD dwCap = 0x0;
+	if (DllSetupDiGetDeviceRegistryProperty(hDevInfo, pspDevInfoData, SPDRP_CAPABILITIES, &DataT, (PBYTE)&dwCap, 4, &nSize)) {
```

Comment:
Why `4` instead of `buffSize` like the other instances we call `DllSetupDiGetDeviceRegistryProperty`?

Reply:
Because we are trying to get a DWORD, so it is exactly 4.

Reply:
Can we use `sizeOf(DWORD)` or something similar?

Reply:
Actually it is defined as;
typedef unsigned long       DWORD;
And will not change in the future.
But no problem we can use sizeof(DWORD).

Why do we use buffSize for all other instances? What's different about this one?
In other instances we are reading data to buf variable. In this instance we are reading data to dwCapabilities which is a DWORD.

Reply:
Done.